central/processlisteningonport/datastore/datastore.go

```diff
@@ -30,6 +31,9 @@ type DataStore interface {
 	WalkAll(ctx context.Context, fn WalkFn) error
 	RemoveProcessListeningOnPort(ctx context.Context, ids []string) error
 	RemovePlopsByPod(ctx context.Context, id string) error
+	PruneOrphanedPLOPs(ctx context.Context, pool postgres.DB, orphanWindow time.Duration) int64
+	PruneOrphanedPLOPsByProcessIndicators(ctx context.Context, pool postgres.DB, orphanWindow time.Duration)
```

Comment:
nit:  Could we simply have one function called PrunePLOPs and then it can prune orphaned plops as well as prune by process indicator?  That way pruning only needs one entry point to the datastore.

Reply:
`PruneOrphanedPLOPsByProcessIndicators` is called by `removeOrphanedProcesses` and `PruneOrphanedPLOPs` is called by `removeOrphanedPLOPs`. I think the reason for this is that `PruneOrphanedPLOPsByProcessIndicators` must be called before `PruneOrphanedProcessIndicators`. To ensure that, they were both put in `removeOrphanedProcesses`. I am open to moving `PruneOrphanedPLOPsByProcessIndicators` to `removeOrphanedPLOPs`.